scripts/fast-reboot

```diff
@@ -171,7 +171,7 @@ function init_warm_reboot_states()
     # If the current running instance was booted up with warm reboot. Then
     # the current DB contents will likely mark warm reboot is done.
     # Clear these states so that the next boot up image won't get confused.
-    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "express-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
         sonic-db-cli STATE_DB eval "
```

Comment:
Do we have sonic-mgmt tests for Cisco express-reboot feature?

Reply:
yes, will send out a PR for express boot sonic-management tests